src/ripple/resource/impl/Logic.h

```diff
@@ -189,42 +194,11 @@ class Logic
         }
 
         m_journal.debug <<
-            "New admin endpoint " << *entry;
+            "New unlimited endpoint " << *entry;
 
         return Consumer (*this, *entry);
     }
 
-    Entry& elevateToAdminEndpoint (Entry& prior, std::string const& name)
```

Comment:
It's been unused for 25 months: https://github.com/ripple/rippled/commit/a1b487c5128e6531295d6089bef474a391256ce4